train.py

```diff
@@ -70,6 +70,10 @@
                         type=int, default=10)
     parser.add_argument('--n-evaluations', help='Number of evaluations for hyperparameter optimization',
                         type=int, default=20)
+    parser.add_argument('--storage', help='Database storage path if distributed optimization should be used', type=str,
```

Comment:
maybe add a check that the `study_name` is not None when passing a storage?

Reply:
Not necessary in my opinion. Optuna will create a unique name in this case ( and will print it to the terminal, thus the user can see it.
`[I 2020-06-22 16:54:51,869] A new study created with name: no-name-5b736a40-d200-40d7-a680-115a33b2358a
`
But if you would like to have a check, I can add it

Reply:
Ok, then please add a warning